.github/workflows/build-deploy.yml

```diff
@@ -43,17 +43,15 @@ jobs:
         env:
           GITHUB_USERNAME: x-access-token
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        if: github.base_ref == 'main' && github.event.pull_request.merged == true
         run: |
           export GIT_COMMIT_HASH=$(git log -n 1 --pretty=format:'%h')
           export GIT_COMMIT_DATE=$(git log -1 --pretty='%ad' --date=format:'%Y%m%d%H%M%S')
-          export VERSION=2.${GIT_COMMIT_DATE}_${GIT_COMMIT_HASH}
+          export VERSION=2.${GIT_COMMIT_DATE}_${GIT_COMMIT_HASH}_token-support-4
```

Comment:
Hva gjør vi med bygging og deploy. Nå som ef ikke kan oppgradere?

Reply:
Jeg synes det var godt forslag du hadde, hvor dette kan være versjon 3 av felles for RestClient, mens restTemplate kan fortsette å være versjon 2 👍